Scripts/import-logs.ps1

```diff
@@ -1,109 +1,259 @@
+
 #########powershell script to upload log files into s3 bucket###########
 
 #s3Bucket path to save the log files
 param (
-    [string] $version,
-    [string] $ReleaseId
+    [Parameter(Mandatory=$true)]
+    [string] $Lansaversion,
+    [Parameter(Mandatory=$true)]
+    [string] $PipelineReleaseId,
+    [Parameter(Mandatory=$true)]
+    [string[]] $databaseTypes
 )
-$Lansaversion = $version
-$PipelineReleaseId = $ReleaseId
+
 $s3BasePath = 'lansa-us-east-1/db-regression-test/logs' 
 $region = 'us-east-1'
- 
-#Total Summary for all Dbs
-$totalSummaryFile = 'C:\Program Files (x86)\LANSA\Verifier_Total_Summary.txt'
-$totalSummaryPath = Join-Path $Lansaversion $PipelineReleaseId
-$totalSummaryKey = $totalSummaryPath + '/Verifier_Total_Summary.txt'
-Write-S3Object -BucketName $s3BasePath -Key $totalSummaryKey -Content $totalSummaryFile 
 
+$s3BaseKey =  $Lansaversion +'/'+ $PipelineReleaseId
 
-################################ list of databases ################################################################
-$databases = New-Object -TypeName 'System.Collections.ArrayList';
- #$databases.Add("SQL Server")
- $databases.Add("AZURESQL")
- $databases.Add("Oracle")
- $databases.Add("MySQL")
- $databases.Add("SQLANYWHERE")
 
 
-############################### writing log files from source to s3 ################################################
 
+Write-Host($db)
+Write-Host('')
+$databases = New-Object -TypeName 'System.Collections.ArrayList';
+Write-Host('list of databases:') 
+Write-Host('--------------------------------------------------------------')
+foreach ($db in $databaseTypes)
+{
+    if($db -ne 'SQL Server')
+    {
+        $databases.Add($db)
+    }  
+    Write-Host($db) 
+}
```

Comment:
The database types are here: https://github.com/lansa/db-regression/wiki#:~:text=Limited%20to%20these,n/a
`SQL Server` is not one of them. Its `MSSQLS`.
You need to map from DB Type to RootPath.